pkg/utils/utils.go

```diff
@@ -76,7 +77,21 @@ func GetPfName(pciAddr string) (string, error) {
 		}
```

Comment:
I think we need to do the same logic for L#76 to support the case where pciAddr is not a VF (i.e it is a PF) and the device eswitch mode is switchdev.